tests/hyperv_test.go

```diff
@@ -267,7 +268,11 @@ var _ = Describe("[Serial][sig-compute] Hyper-V enlightenments", Serial, func()
 			Expect(err).ToNot(HaveOccurred(), "Should get VMI")
 			Expect(vmi.Spec.Domain.Features.Hyperv.EVMCS).ToNot(BeNil(), "evmcs should not be nil")
 			Expect(vmi.Spec.Domain.CPU).ToNot(BeNil(), "cpu topology can't be nil")
-			pod := libvmi.GetPodByVirtualMachineInstance(vmi, vmi.Namespace)
+			Eventually(func() error {
```

Comment:
I think you need to wait for the correct phase/condition and only then search for the pod.

Reply:
Hey Thanks for reviewing.
Done